core/src/main/java/com/nisovin/magicspells/spells/PlayerMenuSpell.java

```diff
@@ -153,12 +161,33 @@ public boolean castFromConsole(CommandSender sender, String[] args) {
 		return true;
 	}
 
+	private ItemStack createItem(String path, String defaultName) {
+		ItemStack item = null;
+		if (isConfigSection(path)) {
+			MagicItem magicItem = MagicItems.getMagicItemFromSection(getConfigSection(path));
+			if (magicItem != null) item = magicItem.getItemStack();
+		} else {
```

Comment:
You shouldn't support magic items from configuration sections here, just strings.